openlibrary/templates/covers/manage.html

```diff
@@ -58,7 +58,7 @@
 
     <div class="formElement" style="margin:40px 0 0 30px;">
         <button type="submit" value="Save" class="largest">$_("Save")</button>
-        <a class="floaterClose close-popup" href="javascript:;" onclick="parent.\$.fn.colorbox.close();">$_("Cancel")</a>
+        <a class="floaterClose floaterShut--parent close-popup">$_("Cancel")</a>
```

Comment:
`close-popup` does the same thing (i.e. `$.fn.colorbox.close()`) (see automatic.js). Let's kill it in favor of your new methods. (Put in a separate commit, bitte)

Reply:
Ditto above.